### ISSUE TITLE
Wechaty v0.20 & Puppet Padchat v0.14 Released. Padchat Alpha Testing Closed.

### ISSUE THREAD

**Issue by zixia** [Chatie/wechaty#1296]
## UPDATE

We had released wechaty@^0.20 with wechaty-puppet-padchat@^0.14 today.

So Alpha Testing will be terminated as we announced before.

Anyone who wants to use [wechaty-puppet-padchat](https://github.com/lijiarui/wechaty-puppet-padchat), please apply padchat token at [BotOrange.com](http://www.botorange.com/)

Ad: If you want to test new puppet implmentation with iPhone App Hook, you can apply for join alpha testers list at [Wechaty Puppet Ioscat Testing: iPhone(iOS) Wechat Hooking Support are comming!](https://github.com/Chatie/wechaty/issues/1530)

---------------

Hello to developers who are using Wechaty,

We have excellent news for you: **Wechaty New Puppet System is going to ready for testing!**, and Alpha Testing Volunteers are open to applications now.

If you want to benefit from all the new features, and cannot wait for the next v0.18 release, then you are welcomed to apply for joining the v0.19 Alpha Tester List!

After HUGE refactoring the Wechaty Framework code base, today we can use the branding new Puppet System to connect Wechaty API to any kinds of Puppets, for example:

1.  [PuppetMock](https://github.com/Chatie/wechaty/tree/ed72a78b61ccc352d9bd9f5a06054a218cdd1d0d/src/puppet-mock)(for mocking)
1. [PuppetWechat4U](https://github.com/Chatie/wechaty/tree/ed72a78b61ccc352d9bd9f5a06054a218cdd1d0d/src/puppet-wechat4u)(Web API via HTTP), and
1. our very classic [PuppetPuppeteer](https://github.com/Chatie/wechaty/tree/ed72a78b61ccc352d9bd9f5a06054a218cdd1d0d/src/puppet-puppeteer)(Web API via Browser)

[![Wechaty](https://chatie.io/wechaty/images/wechaty-logo-en.png)](https://github.com/chatie/wechaty)

Start from today; we need some volunteers who are interested in joining the alpha testing, for example, our new Padchat Wechat Puppet which backed by the iPad Protocol. If you are one of them, please leave a comments under this issue,  introduce yourself about who you are and what you want to do(please DO follow the following TEMPLATE and fill it in details, or you might not be able to be selected), so that we can be able to add you to the alpha testers list if you are qualified.

### Wechaty Puppet Alpha Tester Apply Template

-----------
### 3. Are Your Bot Related with the Following Tasks? (YES/NO)

1. Gambling
2. Adault & Sex

### 7. I understand that the Wechaty v0.17 is in very _Early Alpha Stage_, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)

-----------

Looking forward to hearing back from you, and please submit a new issue if you found any bug or have any feature requests for Wechaty. PR is also welcome with an issue submitted first.

At last, I'd like to say thank you to @lijiarui, @binsee, and @linyimin, who are the main contributors to the puppet system. v0.17 will not possible to go out without your help; it's our teamwork!

Have a good weekend!

Best,
Huan

### Update Jun 24, 2018

Hi all, Wechaty Alpha Tester List current has 11 developers before we stepped into the new developing version v0.17. We will continue accepting alpha tester application until the list reaches to 17 testers. After that, we will queue all the new requests, and start our qualification program to measure the alpha testers every week. If a tester cannot meet the requirement of our qualification program, they will be removed from the alpha tester list, and then we will be able to accept the new application from the queue again.

**Comment by zixia** [comment]
@sammychiang Thanks for your application, and welcome to join the Wechaty Alpha Tester List! 

You had just been invited to the Wechaty Alpha Tester List Wechat Room, there will be the 2nd place for discussion between all the testers other than Github. Please be prepared to meet all kind of bugs, and good luck! :smiling_imp:

**Comment by alamusi** [comment]
@zixia Thanks for the invitation. count my team in.

Introduce Yourself(who you are): founder of Canadian Startup in AI, Mobile, and Cloud Service.

Explain the Problem You Want to Solve:(what you do): Conversational UI in general. WeChat is one platform we can't ignore. WebPuppet has many limitations and quite problematic. I am excited to see other form of puppets becoming available! 

PS: is there an option to self host padchat? my client would like to avoid the risk of relying on young third party service providers, so they prefer to have my team build infrastructure and operate the cloud service.

Your Email: alamusi@questflex.com

Your Wechat: alamusi2003

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support.: YES

**Comment by zixia** [comment]
@alamusi Thanks for the application, and I'm glad to know that you are an entrepreneur with AI and ChatBot! I had added your Wechat and please accept so that I can invite you to the Alpha Tester room, and we will discuss the future steps over there.

For your question: _is there an option to self-host padchat_, the short answer is: **YES**

The long answer: Wechaty did not provide the iPad Protocol by itself, it only provides an Abstract Puppet Interface to connect to the iPad Protocol, which means we have to have a service provider for it.

At the same time, it is possible to have a private deploy of that. From our current alpha stage testing, we will focus on pushing the Puppet code from alpha to beta. For long-term, I'd like to suggest you talk with @lijiarui in business, she is the founder of  BotOrange, and BotOrange is a Beijing Startup in AI and Chatbot.

**Comment by xinbenlv** [comment]
Singup!

**Comment by lvxuan263** [comment]
1. Introduce Yourself(who you are): developer of a startup

2. Explain the Problem You Want to Solve:(what you do): Message listening for Some companies

3. Your Email: lvxuan263@sina.com

4. Your Wechat: bei_188699905

5. I Had Joined the Wechaty Developers' Home Wechat Room: YES

6. I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support:  YES

**Comment by zmrbak** [comment]
1. Introduce Yourself(who you are):
大学教师，计算机工作者，计算机编程爱好者。

2. Explain the Problem You Want to Solve:(what you do):
我现在在运营一些列微信群，群中有一些讲座，我通过WeChaty控制一系列微信号，在群中互动。给予Web协议的WeChaty在号上有一些限制，如果能使用 IPAD协议的话，这些限制就可以突破。所以准备尝试新协议。

3. Your Email:
20995658@qq.com

4. Your Wechat:
zqm028a

5. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

6. I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here)
YES

**Comment by zixia** [comment]
@xinbenlv @lvxuan263 @zmrbak Awesome! Welcome to join and please feel free to report bugs on Github and discuss in Wechat Alpha Test Room.

I had just sent email to you with the necessary information of how to start your Alpha Testing with v0.15, please check your email address and let me know you had received it.

Please also add my wechat 918999 so that I could be able to invite you to the Wechaty Alpha Tester room, happy testing!

**Comment by shitiven** [comment]
1. Introduce Yourself(who you are): developer leader of a startup

2. Explain the Problem You Want to Solve:(what you do): Provide convenient operation tools for internal staff through WeChat.

3. Your Email: fackweb@gmail.com

4. Your Wechat: shitiven

5. I Had Joined the Wechaty Developers' Home Wechat Room: NO

6. I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by radishyell** [comment]
1. Introduce Yourself(who you are): radih_huang  programmer
2. Explain the Problem You Want to Solve:(what you do): get user wechatid，send video message
3. Your Email: 597395959@qq.com
4. Your Wechat:bboyct597395959

**Comment by zixia** [comment]
@shitiven please confirm that you had joined the developers home before we can continue to process your apply, thanks.

**Comment by gxqc** [comment]
Introduce Yourself(who you are): developer leader of a startup

Explain the Problem You Want to Solve:(what you do): Provide convenient system of WeChat.

Your Email: gxsjtu@msn.com

Your Wechat: guoxuan_private

I Had Joined the Wechaty Developers' Home Wechat Room: NO

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by zixia** [comment]
@gxqc Please [join the developers home](https://github.com/Chatie/wechaty#join-us) before we can continue to process your application, thanks.

**Comment by zixia** [comment]
@radishyell Please check you email for the next steps of joining the Alpha Tester List, thanks.

**Comment by hello-nick** [comment]
Introduce Yourself(who you are): leader of a e-commerce team

Explain the Problem You Want to Solve:(what you do): Provide sales tools for e-commerce through WeChat.

Your Email: 58333743@qq.com

Your Wechat: nick_shcn

I Had Joined the Wechaty Developers' Home Wechat Room: NO

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by hirohe** [comment]
1. an web developer

2. plain to use WeChaty for automatic message transport

3. email: hesreg#hotmail.com

4. wechat id: iamhes

5. I Had Joined the Wechaty Developers' Home Wechat Room, YES, the second wechaty group

6. I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by zixia** [comment]
@hello-nick Your application had been approved.

**Comment by zixia** [comment]
@hirohe Your application had been approved, please check your email and follow the steps to join the Alpha Tester List Wechat Room.

**Comment by jlisbz** [comment]
Hi @zixia,

Here is my application for alpha testing.

1. Introduce Yourself(who you are): Developer and Cyber Security Consultant from Toronto, Canada
2. Explain the Problem You Want to Solve:(what you do): To manage my wechat groups for professional groups
3. Your Email:   jli@moresecure.ca
4. Your Wechat:  JohnJunLi
5. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)  YES
6. I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here) YES - Jun Li

Looking forward to the more robust technology.

Thanks a lot.

JL

**Comment by zixia** [comment]
@jlisbz Welcome, please check your email for the next steps of the application.

**Comment by Madao-3** [comment]
Introduce Yourself(who you are): leader of a e-commerce team

Explain the Problem You Want to Solve:(what you do): Provide sales tools for e-commerce through WeChat.

Your Email: chrishyman256@gmail.com

Your Wechat: madao3

I Had Joined the Wechaty Developers' Home Wechat Room: YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by hxllyl** [comment]
Introduce Yourself(who you are): developer of a e-commerce team

Explain the Problem You Want to Solve:(what you do): Provide sales tools for e-commerce through WeChat.

Your Email: huxiaolong100@126.com

Your Wechat: hxllyl

I Had Joined the Wechaty Developers' Home Wechat Room: YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by grepug** [comment]
1. Introduce Yourself(who you are): founder of a startup based on Wechat services
2. Explain the Problem You Want to Solve:(what you do): to provide offline sport activities registration in wechat groups, along with our wechat miniprogram, for a better registration experience.
3. Your Email: grepug@163.com
4. Your Wechat: grepug
5. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here) YES
6. I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here) YES

**Comment by kis87988** [comment]
1. Introduce Yourself(who you are): developer of a student group
2. Explain the Problem You Want to Solve:(what you do): wechat groups, and manage group. send some info to student and stats analysis
3. Your Email: kis87988@gmail.com
4. Your Wechat: kis87988
5. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here) YES
6. I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here) YES

**Comment by xxxxlr** [comment]
Introduce Yourself(who you are): developer
Explain the Problem You Want to Solve:(what you do): get the wxid for a better and accurate search result
Your Email: xxxxlr@gmail.com
Your Wechat: wxid_smpwqcsrzmat22
I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here) YES
I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here) YES

**Comment by zixia** [comment]
@grepug @kis87988 @xxxxlr Welcome! 

Your application had been approved, please check your email for further steps to join the Alpha Tester List.

**Comment by as181920** [comment]
Introduce Yourself(who you are): developer
Explain the Problem You Want to Solve:(what you do): message proxy to CRM
Your Email: as181920@gmail.com
Your Wechat: as181920

I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here) YES
I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here) YES

**Comment by weibo6631352** [comment]
Introduce Yourself(who you are): software developer

Explain the Problem You Want to Solve:(what you do): Message listening for  chatRoom

Your Email: wwb-WeiBo.Wang@outlook.com

Your Wechat: qq838524022

I Had Joined the Wechaty Developers' Home Wechat Room: YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by bpgoing** [comment]
1. Introduce Yourself(who you are): developer

2. Explain the Problem You Want to Solve:(what you do): Message manage

3. Your Email: zopxygoing@gmail.com

4. Your Wechat: limsink

5. I Had Joined the Wechaty Developers' Home Wechat Room: YES

6. I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by zixia** [comment]
@as181920 @Hunk-zhurui @weibo6631352 @bpgoing Your applications are approved. Please check your email inbox for the next steps. Welcome!

**Comment by zixia** [comment]
### Update Jun 24, 2018

Hi all, Wechaty Alpha Tester List current has 11 developers before we stepped into the new developing version v0.17. We will continue accepting alpha tester application until the list reaches to 17 testers. After that, we will queue all the new requests, and start our qualification program to measure the alpha testers every week. If a tester cannot meet the requirement of our qualification program, they will be removed from the alpha tester list, and then we will be able to accept the new application from the queue again.

**Comment by qcliu** [comment]
1. Introduce Yourself(who you are): I'm the tech head of a start-up project 'Jiepx.com'.
2. Explain the Problem You Want to Solve:(what you do): Filter user's message and reply accordingly.
3. Your Email: qingcao@vip.qq.com
4. Your Wechat: qingcao-liu
5. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)  YES
6. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here) YES, Qingcao Liu

**Comment by 10822123** [comment]
I want to join the test
I want to solve the multi-WeChat multi-user business operation
my name :mingliang.zhang
email:10822123@qq.com
My wechat 18802000503

thank you

**Comment by zhoumh1988** [comment]
@zixia 
Introduce Yourself(who you are): 前端经理

Explain the Problem You Want to Solve:(what you do): get user id，send voice message, keywords reply and more. 

Your Email: zhoumh1988@163.com

Your Wechat: LittleStrongZhou

I Had Joined the Wechaty Developers' Home Wechat Room: YES

I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by Neil-UWA** [comment]
1. Introduce Yourself(who you are): I'm Neil, a software engineer
2. Explain the Problem You Want to Solve:(what you do): We would like to provide a stable wechat robot for our clients to manage their wechat groups. As long as the webot robot is reliable and be to able to scale, we can pay reasonable service fee if necessary.
3. Your Email: lvjinang@yahoo.com.au
4. Your Wechat: _Neil_Lu_
5. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: Yes
6. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: Yes

**Comment by Arvintian** [comment]
Introduce Yourself(who you are): software developer

Explain the Problem You Want to Solve:(what you do): Integrate wechaty to work flow.

Your Email: arvintian8@163.com

Your Wechat: tian17865933200

I Had Joined the Wechaty Developers' Home Wechat Room: YES

I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by zixia** [comment]
@qcliu @zhoumh1988 @Neil-UWA @Arvintian Your applications are approved. Please check your email inbox for the next steps. Welcome!

**Comment by wuying8** [comment]
Introduce Yourself(who you are):
微信群产品开发负责人。

Explain the Problem You Want to Solve:(what you do):
主要负责微信群产品的开发，例如群直播之类的活动、成员互动等，由于web版协议的微信提供的服务不是很稳定，加上语音效果很差，所以想尝试一下pad协议的效果，获得更佳质量的语音效果

Your Email:
86824400@qq.com

Your Wechat:
wuying8

I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here)
YES

**Comment by Zhou-Bill** [comment]
Introduce Yourself(who you are): developer

Explain the Problem You Want to Solve:(what you do):  i have  use wechaty for web to build a bot, but  Too much limit and my account was sealed, i want to use padchat to solve my problem

Your Email: 735051883@qq.com

Your Wechat: 735051883

I Had Joined the Wechaty Developers' Home Wechat Room: YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by lijiarui** [comment]
@wuying8 @Zhou-Bill Your applications are approved. Please check your email inbox for the next steps. Welcome!

**Comment by xiangye5941** [comment]
Introduce Yourself(who you are): developer

Explain the Problem You Want to Solve:(what you do): Wechat group management tools.

Your Email: nb@lijunxiang.com

Your Wechat: imLiJunXiang

I Had Joined the Wechaty Developers' Home Wechat Room: YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by xiangye5941** [comment]
1. Introduce Yourself(who you are): 
Hi! I'm Justin, a developer.
2. Explain the Problem You Want to Solve:(what you do):
Wechat group management tools.
3. Your Email: nb@lijunxiang.com
4. Your Wechat: imLiJunXiang
5. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)
Yes.
6. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
Yes, by Justin.

**Comment by cxj110** [comment]
1. Introduce Yourself(who you are): 微信群运营产品技术负责人
2. Explain the Problem You Want to Solve:(what you do): 需要大批量的非Web版机器人
3. Your Email: xuejin.chen@gemii.cc
4. Your Wechat: cxj_hit
5. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: Yes
6. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES, X.J Chen

**Comment by zhaokefei** [comment]
1. Introduce Yourself(who you are): I'm kefei, a developer
2. Explain the Problem You Want to Solve:(what you do): Wechat group management tools.
3. Your Email: kefei.zhao@gemii.cc
4. Your Wechat: zhao546109656
5. I Had Joined the Wechaty Developers' Home: YES
6. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here): YES, kefei

**Comment by rabbitorange** [comment]
@zhaokefei  Your applications are approved. Please check your email inbox for the next steps. Welcome!

**Comment by zixia** [comment]
### Update Notice

Padchat Alpha Testing will be terminated when we had been ready for Beta Testing.

The date is still not decided but it will not take too long from now.

**Comment by zixia** [comment]
@solarhell not yet, but we hope soon.

**Comment by yiqun1010** [comment]
Introduce Yourself(who you are): hi! I'm a developer.
Explain the Problem You Want to Solve:(what you do): wechat robot management for taobao coupons
Your Email: 56828390@qq.com
Your Wechat: yiqun1010
I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: Yes
I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: Yes

**Comment by caicaicai** [comment]
1. Introduce Yourself(who you are):
 I'm caicaicai, a developer
2. Explain the Problem You Want to Solve:(what you do):
wechat robot management for my server
3. Your Email:
admin@xiaocaicai.com
4. Your Wechat:
Cai_Zhongzhen
5. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: 
YES
6. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support:
YES

**Comment by jqs7** [comment]
1. Introduce Yourself(who you are): I'm Jqs7, a full stack developer
2. Explain the Problem You Want to Solve:(what you do): wechat robot used for development (CI/Issue management), the wechat robot base on web api is not so stable.
3. Your Email: 7@jqs7.com
4. Your Wechat: Jiaqi_Se7en
5. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: YES
6. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support:  YES

**Comment by feeloc** [comment]
1. Introduce Yourself(who you are): a web developer, use Wechaty for long
2. Explain the Problem You Want to Solve:(what you do): test new Wechaty, so I can use it to my system smooth
3. Your Email: hjm1989@gmail.com
4. Your Wechat: feeloc

**Comment by hbk671104** [comment]
1. Introduce Yourself (who you are): I'm Bokang (BK), a software engineer from a renowned token fund

2. Explain the Problem You Want to Solve: (what you do):
I'm trying to solve the issue that investment managers often need to reply a message of same content, voice message particularly, to multiple wechat groups. Wechaty is able to empower them with the capability of forwarding message to multiple other groups in a nick of time, thus greatly saving the cost of communication.

3. Are Your Bot Related with the Following Tasks? NO
Gambling
Adult & Sex

4. Your Email: hbk671104@gmail.com

5. Your Wechat: hbk671104

6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)

YES

7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)

YES

**Comment by zixia** [comment]
### Update

This application program will be terminated after we published v0.20

**Comment by ekousp** [comment]
1. Introduce Yourself(who you are): **tianzhao**

2. Explain the Problem You Want to Solve:(what you do): **dev at a startup. need to reply a lot msgs in the customer group**

3. Are Your Bot Related with the Following Tasks? (YES/NO) **NO**
Gambling
Adault & Sex
4. Your Email: **ekousp@gmail.com**
5. Your Wechat: **ekousp**
6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here) **YES**
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here) **YES tianzhao**

**Comment by lijiarui** [comment]
@linhaobin @yiqun1010 @caicaicai @jqs7 @hbk671104 @ekousp Your applications are approved. Please check your email inbox for the next steps. 

Welcome!

**Comment by 3014zhangshuo** [comment]
1.Introduce Yourself(who you are): 
张硕 rails developer。

2.Explain the Problem You Want to Solve:(what you do):
Automatically receive user resume files

3.Are Your Bot Related with the Following Tasks? NO
  Gambling
  Adult & Sex

4.Your Email:
shuo@wondercv.com

5.Your Wechat:
QMXZDDD

6.I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

7.I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here)
YES

**Comment by stxxnet** [comment]
1.Introduce Yourself(who you are):
I'm  a FE developer
2.Explain the Problem You Want to Solve:(what you do):
wechat robot management for personal taoke
3.Your Email:
cnjie@outlook.com
4.Your Wechat:
kanvip2018
5.I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md:
YES
6.I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support:
YES

**Comment by solarhell** [comment]
1.Introduce Yourself(who you are):
solarhell (backend developer)

2.Explain the Problem You Want to Solve:(what you do):
Manage groups programmingly.

3.Are Your Bot Related with the Following Tasks? NO
Gambling
Adult & Sex

4.Your Email:
songjiaxin2008@gmail.com

5.Your Wechat:
CrsPrince

6.I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

7.I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here)
YES

**Comment by lijiarui** [comment]
@3014zhangshuo @solarhell Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by fengzie** [comment]
1.Introduce Yourself(who you are):
fengzie (A startup owner)

2.Explain the Problem You Want to Solve:(what you do):
Create a bot for user, user can forward content from other applications to the bot and save them.

3.Are Your Bot Related with the Following Tasks? NO
Gambling
Adult & Sex

4.Your Email:
526567244@qq.com

5.Your Wechat:
mulgore_wx

6.I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

7.I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here)
YES

**Comment by kaffa** [comment]
Wechaty Puppet Alpha Tester Apply Template

1. Introduce Yourself(who you are):
```
  10+ years programming experience with testing knowledge and skills
```
2. Explain the Problem You Want to Solve:(what you do):
```
  To analyze the text, documents in the wechat. 
  To build a wechat robot or a personal assistant
```
3. Are Your Bot Related with the Following Tasks? (YES/NO)
Gambling
Adault & Sex
```
  No
```
4. Your Email:
```
  kaffacoffee@outlook.com
```
5. Your Wechat:
```
  initiz.replace('z', 'l')
```
6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md:** **(Please join the group if you are not, and then say YES at here)
```
  YES
```
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
```
  YES
```

**Comment by pkjy** [comment]
1. Introduce Yourself(who you are): FE developer 

Explain the Problem You Want to Solve:(what you do): Create a assistant bot for myself.

Your Email: pkjy@outlook.com

Your Wechat: tnqe214161398

I Had Joined the Wechaty Developers' Home Wechat Room: YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by Pengzw** [comment]
Introduce Yourself(who you are):
计算机工作者，计算机编程爱好者。

Explain the Problem You Want to Solve:(what you do):
我现在游戏公司开发游戏, 需要微信群推广业务以及和微信用户互动等，由于web版协议的微信提供的服务不是很稳定，所以特别希望能用上ipad协议, 期望能解决web协议的问题

Your Email:
1273230678@qq.com

Your Wechat:
pngzws

I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here)
YES

**Comment by cameron93lin** [comment]
1. Introduce Yourself(who you are):
Computer Science student.
2. Explain the Problem You Want to Solve:(what you do):
I am trying to create a bot which can answer new student questions base existed new student guide (www.unlguide.com) and manage new student wechat group. 
3. Are Your Bot Related with the Following Tasks? (YES/NO)
Gambling
Adault & Sex

NO
4. Your Email: cameron93lin@gmail.com
5. Your Wechat: mike93star
6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: 
YES
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: 
YES

**Comment by Chlerry** [comment]
**1. Introduce Yourself(who you are):**
Senior Computer Science student at UNL, and full-time Intern at Fiserv.
**2. Explain the Problem You Want to Solve:(what you do):**
I am working with @cameron93lin on a student guide bot. 
**3. Are Your Bot Related with the Following Tasks? (YES/NO) Gambling, Adault & Sex**
NO
**4. Your Email:** sherryandchloe@gmail.com
**5. Your Wechat**: daniel2142
**6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md:**
YES
**7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support:**
YES

**Comment by 764600368** [comment]
1. Introduce Yourself(who you are):
Programmer interested in robots
2. Explain the Problem You Want to Solve:(what you do):
Web based robot does not work anymore.
3. Are Your Bot Related with the Following Tasks? (YES/NO) Gambling, Adault & Sex
NO
4. Your Email: 764600368@qq.com
5. Your Wechat: sailan10
6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md:
YES
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support:
YES

**Comment by ftfuntjh** [comment]
1.Introduce Yourself(who you are):
大学生

2.Explain the Problem You Want to Solve:(what you do):
我现在在运用微信做一些群内自动响应的服务，包括自动弹出附近的美食。

3.Your Email:
768181755@qq.com

4.Your Wechat:
mr_gy-

5.I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here)
YES

**Comment by zhaizong** [comment]
@fengzie Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by zhaizong** [comment]
@fengzie @pkjy @Pengzw @cameron93lin @Chlerry @764600368 @ftfuntjh  Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by BetaSu** [comment]
1.Introduce Yourself(who you are):
frontend engineer

2.Explain the Problem You Want to Solve:(what you do):
to help myself reply some msg

3.Your Email:
313439271@qq.com

4.Your Wechat:
yinyangpamengdong

5.I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here)
YES

**Comment by JinksPeng** [comment]
1.Introduce Yourself(who you are):    I'm Jinks, a software engineer
2.Explain the Problem You Want to Solve:(what you do):  a wechat robot 
3.Your Email:    jinkspeng@gmail.com
4. Your Wechat:  Jinks
5.I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: Yes
I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: Yes

**Comment by iching-long** [comment]
1. Introduce Yourself(who you are):
Programmer interested in wechat robots
2. Explain the Problem You Want to Solve:(what you do):
Web based robot does not work anymore for new wechat account. I want to build a robot for do some automation tasks
3. Are Your Bot Related with the Following Tasks? (YES/NO) Gambling, Adault & Sex
NO
4. Your Email: ichilong@qq.com
5. Your Wechat: Lifedriver
6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md:
YES. the second one
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support:
YES

**Comment by notech** [comment]
1. Introduce Yourself(who you are):
**I'm notech, a web program developer and a leader of a startup team**
2. Explain the Problem You Want to Solve:(what you do):
**i am trying to make a robot which have some features with financial**
3. Are Your Bot Related with the Following Tasks? (YES/NO)

Gambling **NO**
Adault & Sex **NO**
4. Your Email:
**996635852@163.com**
5. Your Wechat:
**cc996635**
6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)
**YES**
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
**YES**  BY notech

**Comment by zhaizong** [comment]
@BetaSu @JinksPeng @iching-long  Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by zhaizong** [comment]
@notech Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by nixuan** [comment]
Introduce Yourself(who you are): 
developer of startup

Explain the Problem You Want to Solve:(what you do):
try to use robot for the group management

Your Email:
nixuan AT live.cn

Your Wechat:
'nathan'.concat('x777')

I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here)
YES by nixuan

Thanks for your work!

**Comment by linxiaojialin** [comment]
1. Introduce Yourself(who you are):
    kiddo,a developer.
2. Explain the Problem You Want to Solve:(what you do):
    Develop an intelligent customer service robot.
3. Are Your Bot Related with the Following Tasks? (YES/NO)Gambling,Adault & Sex
    NO
4. Your Email:
    kiddolin@qq.com
5. Your Wechat:
    j752322428
6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md:(Please join the group if you are not, and then say YES at here)
    YES
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
    YES.kiddo

**Comment by AlphaFase** [comment]
Introduce Yourself(who you are):
    计算机工作者，计算机编程爱好者。

    Explain the Problem You Want to Solve:(what you do):
    我现在在运营一些列微信群，群是优ku, ai奇异，疼讯之类的vip电影分享，我通过WeChaty控制一系列微信号，在群中互动。给予Web协议的WeChaty在号上有一些限制，如果能使用 IPAD协议的话，这些限制就可以突破。所以准备尝试新协议。

    Your Email:
    605608886@qq.com

    Your Wechat:
    weixin605608886

**Comment by apiaoqzh** [comment]
1. Introduce Yourself(who you are):
   A tester also a web,node,wechat miniprogram developer.living in canton, china.first name is 丘, and you can call me 'apiao', 阿飘
2. Explain the Problem You Want to Solve:(what you do):
   I wanna find some methods to manage a number of weixin group in our Company's Project.
   旨在为运营人员提供可靠且高效的社群管理解决方案.
   希望突破web上的限制,使用ipad的协议可以解决一些社群管理的问题.
3. Are Your Bot Related with the Following Tasks? (YES/NO)
   
Gambling NO
Adault & Sex NO
4. Your Email:
565607389@qq.com
5. Your Wechat:
qiuzehang
6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)
YES,i've join the group by the QRCode
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
yes qiuzehang

**Comment by zhaizong** [comment]
@AlphaFase please look at the wechaty puppet alpha tester apply template again, before we can continue to process your apply, thanks.

**Comment by hhxx9922** [comment]
1.Introduce Yourself(who you are):
大学生

2.Explain the Problem You Want to Solve:(what you do):
我现在在运用微信做一些群内自动响应的服务，包括自动弹出附近的球场。

3.Your Email:
15213175654@163.com

4.Your Wechat:
xingxio007

5.I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here)
YES

**Comment by zhaizong** [comment]
@hhxx9922 Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by anatolychelyshev** [comment]
1. Introduce Yourself(who you are): developer leader of a startup

2. Explain the Problem You Want to Solve:(what you do): I need to have access to conversations

3. Are Your Bot Related with the Following Tasks?
Gambling: NO
Adault & Sex: NO

4. Your Email: 

5. Your Wechat: anatoly_wechat

6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)
YES

7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
YES

**Comment by zhaizong** [comment]
@anatolychelyshev please fill in the Email

**Comment by zhaizong** [comment]
@Flyphy Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by yewjia96** [comment]
1. Introduce Yourself(who you are):
Startup Developer

2. Explain the Problem You Want to Solve:(what you do):
2018 new create account issue.

3. Are Your Bot Related with the Following Tasks? (YES/NO)
Nope

4. Your Email:
admin@leonnlooi.com
5. Your Wechat:
leonnlooi
6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)
yes
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
yes. Leonn Looi

**Comment by fxy060608** [comment]
1.Introduce Yourself(who you are):
Front End Developer

2.Explain the Problem You Want to Solve:(what you do):
Web API can not invite members to room since 2018.

3.Are Your Bot Related with the Following Tasks? (YES/NO)
NO

4.Your Email:
fxy060608@gmail.com

5.Your Wechat:
fxy060608

6.I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)
YES

7.I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
YES

**Comment by horiebin** [comment]
Introduce Yourself(who you are):
Startup Developer

Explain the Problem You Want to Solve:(what you do):
2018 new create account issue.

Are Your Bot Related with the Following Tasks? (YES/NO)
Nope

Your Email:
horiebin@foxmail.com

Your Wechat:
horiebin

I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)
yes

I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
yes. Horiebin

**Comment by cyyuen** [comment]
1. Introduce Yourself(who you are): Developer of a startup
2. Explain the Problem You Want to Solve:(what you do): Send customized message to users 
3. Are Your Bot Related with the Following Tasks? (YES/NO) NO
Gambling
Adault & Sex
4. Your Email: colsy2@gmail.com
5. Your Wechat: dotcy_
6. I Had Joined the Wechaty Developers' Home Wechat Room: YSE 
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES. Siyan

**Comment by zhaizong** [comment]
@yewjia96 Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by zhaizong** [comment]
@cyyuen @horiebin @fxy060608 Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by hzz780** [comment]
#### 1. Introduce Yourself(who you are):
Has been Sr. Software Engineer in Baidu.
A developer of a startup.

#### 2. Explain the Problem You Want to Solve:(what you do):
Provide convenient system of WeChat.
Try to reduce the cost of community management.

#### 3. Are Your Bot Related with the Following Tasks? (YES/NO)
Gambling
Adault & Sex
No

#### 4. Your Email:
hzz780@qq.com

#### 5. Your Wechat:
hzz780

#### 6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)
Yes

#### 7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
Yes

huangzongzhe

**Comment by Yongke** [comment]
1. Introduce Yourself(who you are):
Developer of a startup company.  

2. Explain the Problem You Want to Solve:(what you do):
Multiple contacts/groups management bot

3. Are Your Bot Related with the Following Tasks? (YES/NO)
Gambling
Adault & Sex

No

4. Your Email:
wangyongke@gmail.com

5. Your Wechat:
yongkewang

6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)
Yes

7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
Yes, 
yongkewang

**Comment by woshahua** [comment]
introduce Yourself(who you are):
cheng zheng ,a developer
Explain the Problem You Want to Solve:(what you do):
Develop a smart chatbot for my son
Are Your Bot Related with the Following Tasks? (YES/NO)Gambling,Adault & Sex
NO
Your Email:
wohangye@gmail.com
Your Wechat:
wxid_d5dstp278dkq12
I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md:(Please join the group if you are not, and then say YES at here)
YES
I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
YES sure!

**Comment by woshahua** [comment]
1. Introduce Yourself(who you are):
cheng zheng ,a developer

2. Explain the Problem You Want to Solve:(what you do):
Develop a smart chatbot for my son

3. Are Your Bot Related with the Following Tasks? (YES/NO)Gambling,Adault & Sex
NO

4. Your Email:
wohangye@gmail.com

5. Your Wechat:
wxid_d5dstp278dkq12

6.I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md:(Please join the group if you are not, and then say YES at here)
YES

7.I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
YES sure!
ChengZheng

**Comment by zhaizong** [comment]
@woshahua Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by shitiven** [comment]
Introduce Yourself(who you are): developer leader of a startup

Explain the Problem You Want to Solve:(what you do): Provide convenient operation tools for internal staff through WeChat.

Your Email: fackweb@gmail.com

Your Wechat: HelloWoder

I Had Joined the Wechaty Developers' Home Wechat Room: YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by JesseWeb** [comment]
Introduce Yourself(who you are): jesse,a developer using wechat.

Explain the Problem You Want to Solve:(what you do): company message forward

Your Email: 395731482@qq.com

Your Wechat: xyzcain

I Had Joined the Wechaty Developers' Home Wechat Room: YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by shitiven** [comment]
Is anybody here to deal with above requests?

**Comment by zixia** [comment]
@shitiven Sorry for no response for the applications for so many days.

The padchat token is distributed by the BotOrange company, if you want to get a quick response, it will be a good idea to @lijiarui or pm her on Wechat for asking about the progress of your application.

However, as far as I know, the Alpha Testing will be end soon, and currently, there had already been lots of the developers who had switched to the Beta & Production, which required to subscribe a paid plan from BotOrange.

And also we are planning to close this Alpha Apply Form soon. 

Hope the above information helps you, please feel free to let me know if you have more questions.

**Comment by zhaizong** [comment]
@shitiven @JesseWeb @fidonium @sheeplin  Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by nabei** [comment]
1. Introduce Yourself(who you are): Na bei  ,   an OP in a small company.
2. Explain the Problem You Want to Solve:(what you do): Though I am an OP,but ,in small company,I'm all-powerful,so I need this to Improve my work efficiency.
3. Are Your Bot Related with the Following Tasks? (YES/NO) NO
Gambling
Adault & Sex
4. Your Email:  zhus@live.com
5. Your Wechat:zhusma
6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here) Yes
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here) Yes Nabei.

**Comment by chatch** [comment]
1. Introduce Yourself(who you are): Decentralised Apps developer building with Stellar, Mobius, Ethereum

2. Explain the Problem You Want to Solve:(what you do): Build a tip bot for Mobius. Similar to this one for telegram: https://github.com/mobius-network/mobius-tipbot-ruby but build it for Wechat/微信 users.

3. Are Your Bot Related with any of the Gambling, Adult/Sex Tasks? NO

4. Your Email: hatch@tuta.io

5. Your Wechat: wxid_39uww6xwmfso22

6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: YES - in the second room.

7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here) Yes Hatch

**Comment by xwxtwd** [comment]
1. Introduce Yourself(who you are):
xwxtwd,a developer
2. Explain the Problem You Want to Solve:(what you do):
add friends to my group's room management bot
3. Are Your Bot Related with the Following Tasks? (YES/NO)
Gambling
Adault & Sex
no
4. Your Email:
xwxtwd@qq.com
5. Your Wechat:
xwxtwd
6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)
yes
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
yes
xwxtwd

**Comment by Mr-Song** [comment]
1. Introduce Yourself(who you are):
i am a developer .
2. Explain the Problem You Want to Solve:(what you do):
control some wechat account to send batch message to our employee
and can let us control wechat by program

3. Are Your Bot Related with the Following Tasks? (YES/NO)
Gambling
Adault & Sex  NO,
4. Your Email:
xingxing1114@hotmail.com
5. Your Wechat:
helloxxing
6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: 
Yes
7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
YES  fatlamb

**Comment by zuosc** [comment]
Introduce Yourself(who you are):
计算机工作者，计算机编程爱好者。

Explain the Problem You Want to Solve:(what you do):
我现在在运营一些列微信群，使用web版本的微信协议在号上有一些限制，如果能使用 IPAD协议的话，这些限制就可以突破。所以准备尝试新协议。

Your Email:
121115970@qq.com

Your Wechat:
Z_zorro

I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here)
YES

**Comment by zhaizong** [comment]
@nabei @chatch @xwxtwd @Mr-Song Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by zhengrenfeng** [comment]
Introduce Yourself(who you are): developer of a startup

Explain the Problem You Want to Solve:(what you do):  I need a robot to manage my customers in wechat 

Your Email: 290878063@qq.com

Your Wechat: zhengrenfengbrave

I Had Joined the Wechaty Developers' Home Wechat Room: YES

I understand that the Wechaty v0.19 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by zhaizong** [comment]
@zuosc @yeyanchao @zhengrenfeng Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by swfbarhr** [comment]
1、Introduce Yourself(who you are): a developer of wechat related

2、Explain the Problem You Want to Solve:(what you do): robot making

3、Are Your Bot Related with the Following Tasks? (YES/NO) --- NO
Gambling
Adault & Sex

4、Your Email: swfbarhr@gmail.com

5、Your Wechat: ttomatt

6、I Had Joined the Wechaty Developers' Home Wechat Room: YES

7、I understand that the Wechaty v0.19 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: YES

**Comment by IGUIZP** [comment]
1. Introduce Yourself(who you are):
A developer of a startup.

2. Explain the Problem You Want to Solve:(what you do):
Provide convenient system of WeChat.
Try to room invited event wit Wechaty.

3. Are Your Bot Related with the Following Tasks? (YES/NO)
Gambling
Adault & Sex
No

4. Your Email:
suihexinde@163.com

5. Your Wechat:
gzp_1992

6. I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please join the group if you are not, and then say YES at here)
Yes

7. I understand that the Wechaty v0.17 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES, then followed by your Name at here)
Yes

zhipeng gui

**Comment by zhaizong** [comment]
@swfbarhr @sofly @IGUIZP Your applications are approved. Please check your email inbox for the next steps.

Welcome!

**Comment by iszxjslove** [comment]
1、Introduce Yourself(who you are):
A developer of a startup

2、Explain the Problem You Want to Solve:(what you do):
I need a robot to manage my customers in wechat

3、Are Your Bot Related with the Following Tasks? (YES/NO) 
NO

4、Your Email: 
228762287@qq.com

5、Your Wechat:
tmallyx

6、I Had Joined the Wechaty Developers' Home Wechat Room: 
YES

7、I understand that the Wechaty v0.19 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: 
YES

**Comment by zixia** [comment]
### Alpha Testing Closed

We had released wechaty@^0.20 with wechaty-puppet-padchat@^0.14 today.

So Alpha Testing will be terminated as we announced before.

Anyone who wants to use [wechaty-puppet-padchat](https://github.com/lijiarui/wechaty-puppet-padchat), please apply padchat token at [BotOrange.com](http://www.botorange.com/)

Ad: If you want to test new puppet implmentation with iPhone App Hook, you can apply for join alpha testers list at [Wechaty Puppet Ioscat Testing: iPhone(iOS) Wechat Hooking Support are comming!](https://github.com/Chatie/wechaty/issues/1530)

**Comment by tobunka-git** [comment]
Introduce Yourself(who you are):
A technical communicator working for a fortune 100

Explain the Problem You Want to Solve:(what you do):
I would like to run a series of wechat groups for the distribution of coupons and online classes with the help of an AI assistant

Your Email:
tobunka@gmail.com

Your Wechat:
technicaltranslator

I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

I understand that the Wechaty v0.15 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support: (Please say YES followed by your Name at here)
YES

**Comment by biground** [comment]
Introduce Yourself(who you are):
A developer of a startup

Explain the Problem You Want to Solve:(what you do):
I need a robot to manage my customers in wechat

Your Email:
cyt4byouth@163.com

Your Wechat:
cytcyg100709

I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
NO

I understand that the Wechaty v0.19 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support:
YES

**Comment by uz5m** [comment]
Introduce Yourself(who you are):
A developer of a startup

Explain the Problem You Want to Solve:(what you do):
I need a robot to manage my customers in wechat and autochat with them

Your Email:
txj_killer@163.com

Your Wechat:
uz5m00

I Had Joined the Wechaty Developers' Home Wechat Room by Scan QRCode from README.md: (Please make sure you had joined the group before you say YES at here)
YES

I understand that the Wechaty v0.19 is in very Early Alpha Stage, and I have the necessary engineering technics to deal with the bugs instead of just asking for support:
YES

**Comment by zixia** [comment]
## 1. HOW TO USE WECHAT IPAD PROTOCOL WITH WECHATY

Anyone who wants to use [wechaty-puppet-padchat](https://github.com/lijiarui/wechaty-puppet-padchat), please apply padchat token at [BotOrange.com](http://www.botorange.com/)

### 2. HOW TO JOIN WECHAT IOS HOOK WITH WECHATY

If you want to test new puppet implmentation with iPhone App Hook, you can apply for join alpha testers list at [Wechaty Puppet Ioscat Testing: iPhone(iOS) Wechat Hooking Support are comming!](https://github.com/Chatie/wechaty/issues/1530)

**Comment by lijiarui** [comment]
Anyone who want apply padchat token, please scan the following QR code to contact the [BotOrange](www.botorange.com):

![image](https://user-images.githubusercontent.com/6419514/45479213-6f88e080-b778-11e8-87f5-b69e8e4bae11.png)
